src/cli/Export.cpp

```diff
@@ -40,23 +40,23 @@ Export::Export()
 
 int Export::executeWithDatabase(QSharedPointer<Database> database, QSharedPointer<QCommandLineParser> parser)
 {
-    TextStream outputTextStream(Utils::STDOUT, QIODevice::WriteOnly);
-    TextStream errorTextStream(Utils::STDERR, QIODevice::WriteOnly);
+    TextStream out(Utils::STDOUT.device());
```

Comment:
@droidmonkey why is this one not defined the same way? Is it because we use `write` instead? (Also, why are we using `write` and not `<<`)

Reply:
Yes because of the write call. We use that because of an encoding bug in QTextStream for UTF-8 output.